source/sgp_mode/SGPWorld.h

```diff
@@ -130,9 +132,12 @@ class SGPWorld : public SymWorld {
   SyncDataMonitor<double> &GetSymStolenDataNode();
   SyncDataMonitor<double> &GetSymEarnedDataNode();
   void SetupTasksNodes();
+  void SetupOutputNodes();
```

Comment:
This one also should be renamed since I assume it's only square output that is tracked